app/Http/Controllers/BorrowerRecordsController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Cart;
+use App\Http\Models\User;
+use App\Http\Models\Book;
+use App\Http\Models\BorrowerRecord;
+use Illuminate\Support\Facades\Auth;
+
+class BorrowerRecordsController extends Controller
+{
+    public function request(Request $request)
+    {
+        $datas = $request->all();
+        if ($datas['start_date'] > $datas['end_date']) {
+            return redirect()->route('bookbag.index')->with([
+                'fail' => trans('page.date'),
+            ]);
+        }
+        $borrower_record = [];
+        $ids = [];
+        $books = session()->get('item');
+        $records = [];
+        foreach ($books as $key => $value) {
+            array_push($ids, $value['id']);
+        }
+        foreach ($ids as $key => $id) {
+            $borrower_record = BorrowerRecord::create([
+                'book_id' => $id,
+                'user_id' => Auth::id(),
+                'start_date' => $datas['start_date'],
+                'end_date' => $datas['end_date'],
+                'status' => config('const.request'),
+            ]);
+        }
```

Comment:
sao lại dùng 2 vòng for thế này? đều là lấy các quyển sách trong session mà, gộp vào 1 vòng lặp thôi.